src/main/scala/org/ergoplatform/nodeView/wallet/persistence/WalletDigest.scala

```diff
@@ -45,13 +78,13 @@ object WalletDigestSerializer extends ScorexSerializer[WalletDigest] {
 
     val walletAssetBalancesSize = r.getUInt().toIntExact
 
-    val walletAssetBalances = mutable.LinkedHashMap.empty[EncodedTokenId, Long]
+    val walletAssetBalances = scala.collection.mutable.LinkedHashMap.empty[EncodedTokenId, Long]
```

Comment:
Seems like replacing LinkedHashMap with Map is going to be breaking change. 
The roundtrip property will be violated, since walletAssetBalances.toMap is not order preserving.

Reply:
Not sure this property is needed, but ok reverted and removed putBox & removeBox

Reply:
I didn't mean to revert back to mutable. WalletDigest is used as a message and hence its lifetime is not so obvious. As result using mutable public field is unsafe. The Scala 2.13 has https://www.scala-lang.org/api/current/scala/collection/immutable/VectorMap.html for this use case. To keep it simple, I suggest to use Seq[(EncodedTokenId, Long)], which is both immutable and good for serialization. And I think this will not be a performance problem until we frequently have more than 1000 tokens in a wallet digest.

Reply:
@aslesarenko but why not to revert ?
it worked, and changing WalletDigest is not related to the PR